apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -0,0 +1,34 @@
+import { type UserAuthResponseDto } from "@git-fit/shared";
+
+import { Table } from "~/libs/components/components.js";
+import { getUserColumns } from "~/libs/helpers/helpers.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useEffect,
+} from "~/libs/hooks/hooks.js";
+import { actions as userActions } from "~/modules/users/users.js";
+
+import styles from "./styles.module.css";
+
+const AccessManagement = (): JSX.Element => {
+	const dispatch = useAppDispatch();
+	const users = useAppSelector(({ users }) => users.users);
+
+	useEffect(() => {
+		void dispatch(userActions.loadAll());
+	}, [dispatch]);
+
+	const columns = getUserColumns();
+	const data: UserAuthResponseDto[] = users;
```

Comment:
```suggestion
	const data: UserAuthResponseDto[] = getUserRows();
```
you need always create a helper for it even if you can just use an object keys

Reply:
I don't understand what the function is for, what exactly is it supposed to do?

Reply:
I think he means that it should look like this:
```ts
const getUserRows = (users: UserGetAllItemResponseDto[]): UserGetAllItemResponseDto[] =>
  users.map(({ name, ... }) => { name, ... });
```
(not sure if I write types correct)
So the point is that this function will decide which fields of users to display in rows (same as `getUserColumns` decide which columns to dispaly)

Reply:
right, you need only to take the keys that you need

Reply:
Good, but then it turns out that we return to the fact that the User type will be needed, which @liza-veis told me to remove
```
import { type UserAuthResponseDto } from "~/modules/users/users.js";

type User = {
    name: string,
    createdAt: string,
}


const getUserRows = (users: UserAuthResponseDto[]): User[] =>
    users.map((user) => ({
      name: user.name,
      createdAt: user.createdAt,
}));

export { getUserRows };
```

Reply:
Should I add this type again and add a mapper?

Reply:
yeah, you need to create a new return type for it